### ISSUE TITLE
Transacting two datoms in the same transaction with the same fulltext string will fail

### ISSUE THREAD

**Issue by rnewman** [mozilla/mentat#565]
Just a bug.

```
mentat=> .t [ {:db/ident :community/category
mentat.>   :db/valueType :db.type/string
mentat.>   :db/cardinality :db.cardinality/many
mentat.>   :db/index true
mentat.>   :db/fulltext true
mentat.>   :db/doc "All community categories"}
mentat.> ]
TxReport { tx_id: 268435457, tx_instant: 2018-02-21T02:19:44.805439Z, tempids: {} }
mentat=> .t [{:community/category "thing"}]
TxReport { tx_id: 268435458, tx_instant: 2018-02-21T02:19:59.762872Z, tempids: {} }
mentat=> .t [{:community/category "thing"}]
TxReport { tx_id: 268435459, tx_instant: 2018-02-21T02:20:02.410929Z, tempids: {} }
mentat=> .t [{:community/category "thing2"}{:community/category "thing2"}]
Error: Error(DbError(Msg("Could not insert fts statements into temporary search table!")), State { next_error: Some(SqliteFailure(Error { code: ConstraintViolation, extended_code: 1299 }, Some("NOT NULL constraint failed: exact_searches.v0"))), backtrace: None }).
```

**Comment by rnewman** [comment]
Fix is in a06e53df180689a56bc7c012ef1bdb025561095d, in #553.

**Comment by rnewman** [comment]
https://github.com/mozilla/mentat/commit/df3cdb5db6f5e52168443ec3185a1aea0a803c07
